### ISSUE TITLE
protolint:disable not working for enums

### ISSUE THREAD

**Issue by germanschnyder-itx** [yoheimuta/protolint#116]
I´m trying to use the disable marker on some proto files but i cannot get it to work on ENUMS rules.
For example, this file:

```
syntax = "proto3";
// protolint:disable ENUM_FIELD_NAMES_ZERO_VALUE_END_WITH
// protolint:disable ENUM_FIELD_NAMES_UPPER_SNAKE_CASE

package test;

import "google/protobuf/wrappers.proto";

option java_package = "com.test.example";
option java_outer_classname = "InfoProto";

/**
 * Some enum
*/
enum Strategy {
  DEFAULT= 0;
  bY_LINE_ID = 1;
  NONE = 2;
}
``` 

will always output these errors:

```
➜  ~ protolint lint example.proto
[example.proto:17:3] EnumField name "bY_LINE_ID" must be CAPITALS_WITH_UNDERSCORES
[example.proto:16:3] EnumField name "DEFAULT" with zero value should have the suffix "UNSPECIFIED"
``` 

I´ve tried with just one disable and I get the same result. I´ve found that other rules do work (e.g.  `protolint:enable MAX_LINE_LENGTH`)

**Comment by PaulSonOfLars** [comment]
Which version are you using? I cannot reproduce this using your example on `0.22.1`.
Do you have a `.protolint.yaml` file currently active?

**Comment by germanschnyder-itx** [comment]
I'm using the version that gets installed when executing `go get github.com/yoheimuta/protolint`:
```bash
 $> protolint version  
protolint version master(latest)
``` 
Right now this is failing without any configuration file.

**Comment by PaulSonOfLars** [comment]
I can't reproduce this on go `1.13.4`, using the github released package (downloaded through `brew`). Do you have any other examples you could provide?
Or do you have access to any other systems you could try this on?

@yoheimuta might be worth making the master branch use the current git tag/hash/date to avoid "master(latest)" where we have no debug info

**Comment by germanschnyder-itx** [comment]
Docker ?

```bash
➜  ~ cat example.proto 
syntax = "proto3";

package test;

import "google/protobuf/wrappers.proto";

option java_package = "com.test.example";
option java_outer_classname = "InfoProto";

enum Foo {
  // protolint:disable:next ENUM_FIELD_NAMES_UPPER_SNAKE_CASE
  firstValue = 0;    // no error
  second_value = 1;  // protolint:disable:this ENUM_FIELD_NAMES_UPPER_SNAKE_CASE
  THIRD_VALUE = 2;   // spits out an error
}
➜  ~ docker run --volume "$(pwd):/workspace" --workdir /workspace yoheimuta/protolint lint example.proto  
[example.proto:12:3] EnumField name "firstValue" must be CAPITALS_WITH_UNDERSCORES
[example.proto:13:3] EnumField name "second_value" must be CAPITALS_WITH_UNDERSCORES
[example.proto:12:3] EnumField name "firstValue" with zero value should have the suffix "UNSPECIFIED"

```

**Comment by PaulSonOfLars** [comment]
Still not getting any errors. Are you sure you there is no .protolint.yaml file in your directory?
![image](https://user-images.githubusercontent.com/17353640/69796811-90472580-11c6-11ea-8bd6-f68a48e2a263.png)

**Comment by germanschnyder-itx** [comment]
Yep, 100% sure :\

**Comment by germanschnyder-itx** [comment]
Just in case, this is the docker image I've got:
`yoheimuta/protolint                                                    latest               a9438e994d0a        4 days ago          22.2MB`

and I've created a new folder with just the example file in it.

**Comment by germanschnyder-itx** [comment]
Just in case, I also run an image deletion (i.e. `docker rmi IMAGE_ID`) and re-run with fresh docker image. Same error.

```bash
docker run --volume "$(pwd):/workspace" --workdir /workspace yoheimuta/protolint lint example.proto
Unable to find image 'yoheimuta/protolint:latest' locally
latest: Pulling from yoheimuta/protolint
89d9c30c1d48: Pull complete 
b1beb0c291f2: Pull complete 
949a61938cad: Pull complete 
Digest: sha256:74a790c408e3ed83159c443903e6ac674e8305954f1f847ceae382573705d050
Status: Downloaded newer image for yoheimuta/protolint:latest
[example.proto:12:3] EnumField name "firstValue" must be CAPITALS_WITH_UNDERSCORES
[example.proto:13:3] EnumField name "second_value" must be CAPITALS_WITH_UNDERSCORES
[example.proto:12:3] EnumField name "firstValue" with zero value should have the suffix "UNSPECIFIED"

```

**Comment by germanschnyder-itx** [comment]
Also, I get the same 3 errors for all the available tags in dockerhub (v0.20.1 up to latest).
And, just in case, if test the _examples/proto files I get tons of errors (that are not present in the README file)

**Comment by germanschnyder-itx** [comment]
Update, just tried to copy and paste your `cat` output and seems to be working :|

**Comment by germanschnyder-itx** [comment]
Got it, my file has ^M$ chars all over ... that seems to be fooling the linter. Is there a way to support these kind of windows-linux files compatibility ?

**Comment by PaulSonOfLars** [comment]
Typical windows snowflake bugs; I'd assumed it wasn't windows when you mentioned Ubuntu 19, but guess theres no escape.

My guess is that the //protolint:disable directive checks everything until \n, so it assumes the \r is part of the rulename you're trying to ignore;
So it tries to ignore "RULENAME\r" instead of "RULENAME".

I'll open a PR for this in a minute.

**Comment by germanschnyder-itx** [comment]
I know.. I'm in Ubuntu but other devs are in windows... 
The weird thing is that some other rules DO work. But, anyway, I'm looking forward to your PR :)
 Thanks !

**Comment by PaulSonOfLars** [comment]
If I'm correct on the nature of the issue, this should now be fixed.

**Comment by germanschnyder-itx** [comment]
Well, lets see. Will let you know as soon as I cant get to test it.

**Comment by PaulSonOfLars** [comment]
My local tests seem to suggest it works now, so thats promising.

**Comment by yoheimuta** [comment]
@germanschnyder-itx Thank you for reaching out!
I shipped fixed v0.22.2.

- ref. https://github.com/yoheimuta/protolint/releases/tag/v0.22.2

@PaulSonOfLars Thank you for investigating and fixing it!

**Comment by yoheimuta** [comment]
> might be worth making the master branch use the current git tag/hash/date to avoid "master(latest)" where we have no debug info

@PaulSonOfLars Thank you for your nice suggestion!

I also wish to include detail information, even when built by manually.
Do you have any good idea about how to achieve it automatically along the way of the release flow, instead of updating info manually every time?

**Comment by germanschnyder-itx** [comment]
Working like charm :) Thanks!
